src/org/voltdb/jsr107/CacheEventConsumer.java

```diff
@@ -106,20 +108,20 @@ public void run() {
             filter = filterFactory.create();
         }
 
-        try {
+        Properties props = new Properties();
+        props.put("bootstrap.servers", kafkaHostnames);
+        props.put("group.id", cacheName + System.currentTimeMillis());
```

Comment:
We might think if using the current time here is the best approach. It's for sure better than using just `cacheName`. However in case of restart of a service using that library I'll lose some of the updates as I'll start with a new group and jump to the latest offset. Ideally we could allow for user to choose its "instanceId" to correlate the old instance and the new instance and start from the point where the old one stopped. 

That solution also wouldn't create too many consumer groups. With the group id based on timestamp we would potentially end up with a lot of idle consumer groups eating up resources. I don't know the details how it's implemented in VoltDB so maybe that's not an issue, but if VoltDB doesn't automatically clean up stale consumer groups, it could eat up some space.

Reply:
Agree it needs to be fixed. Could we have it look for an environment variable called "VOLT_CACHE_NAME" and use Volt if none found?

Reply:
Sure, that sounds like a good solution, I'll do that.